### PR TITLE
Fix for imagej relative path handling

### PR DESCRIPTION
**Pull request by bernt-matthias**
fixes https://github.com/bgruening/galaxytools/issues/1452. Unfortunately I could not reproduce the problem in a planemo test, bot the change implemented in https://github.com/bgruening/galaxytools/pull/1460/commits/a2d0e956125e1fb924cf8e139560de122b0cf006 fixed the problem on my instance. 

TODO: 

- [x] error log handling
- [x] profile
- [x] tool version macros

I won't update the tools here, since this does not work out of the box

### CODE REVIEW

File: tools/image_processing/imagej2/imagej2_macros.xml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <macros>
-    <token name="@WRAPPER_VERSION@">3.0</token>
+    <token name="@TOOL_VERSION@">20170530</token>
+    <token name="@VERSION_SUFFIX@">20170530</token>
```
Comment (bgruening) [inline tools/image_processing/imagej2/imagej2_macros.xml:3]: This looks wrong. But the old .1 was also not good, so no idea what is bigger.

Comment (bernt-matthias) [reply tools/image_processing/imagej2/imagej2_macros.xml:3]: I just like it if the tool version matches the version of the underlying software

```python
>>> from packaging.version import Version
>>> Version("20170530+galaxy0") > Version("3.0.1")
True
```

Comment (bgruening) [reply tools/image_processing/imagej2/imagej2_macros.xml:3]: Bu the suffix is the same as the version?

Comment (bernt-matthias) [inline tools/image_processing/imagej2/imagej2_macros.xml:3]:
```suggestion
    <token name="@VERSION_SUFFIX@">0</token>
```

**Comment by bgruening** [comment]
Merci!

**Pull request merged**